test/dlc_tests/container_tests/bin/benchmark/get_pytorch_performance_num.py

```diff
@@ -0,0 +1,11 @@
+import argparse
```

Comment:
Not sure if this should be a sepate file. If we're creating a separate file - we should create a generic 'value checker file' that could be used across different benchmarking test. But then again - such a function is better located in conftest or utils, depending on relevance to pytest.

Reply:
For the 1st question, I think your suggestion works. However if we need to use other AMIs located in us-east-1 for other tests, we need to change this line another time.

Reply:
1st parameterizing issue will be resolved in next commit

Reply:
For issue 3, there is another python helper function script located in this folder for post-processing. I might work in a separate PR for them at the same time